acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/BellatrixMergeTransitionAcceptanceTest.java

```diff
@@ -0,0 +1,82 @@
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance;
+
+import static tech.pegasys.teku.test.acceptance.dsl.BesuDockerVersion.DEVELOP;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.infrastructure.time.SystemTimeProvider;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.BesuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNode;
+import tech.pegasys.teku.test.acceptance.dsl.tools.deposits.ValidatorKeystores;
+
+public class BellatrixMergeTransitionAcceptanceTest extends AcceptanceTestBase {
+  private final String NETWORK_NAME = "less-swift";
+
+  private final SystemTimeProvider timeProvider = new SystemTimeProvider();
+  private BesuNode eth1Node;
+  private TekuNode tekuNode;
+
+  @BeforeEach
+  void setup() throws Exception {
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    eth1Node = createBesuNode(DEVELOP, this::configureBesuNode);
+    eth1Node.start();
+
+    final int TOTAL_VALIDATORS = 4;
+    final ValidatorKeystores validatorKeystores =
+        createTekuDepositSender(NETWORK_NAME).sendValidatorDeposits(eth1Node, TOTAL_VALIDATORS);
+
+    tekuNode =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, genesisTime)
+                    .withDepositsFrom(eth1Node)
+                    .withStartupTargetPeerCount(0)
+                    .withValidatorKeystores(validatorKeystores)
+                    .withValidatorProposerDefaultFeeRecipient(
+                        "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73")
+                    .withExecutionEngineEndpoint(eth1Node.getInternalEngineJsonRpcUrl()));
+    tekuNode.start();
+  }
+
+  @Test
+  void shouldHaveNonDefaultExecutionPayloadAndFinalizeAfterMergeTransition() {
+    tekuNode.waitForGenesis();
+    tekuNode.waitForLogMessageContaining("MERGE is completed");
+    tekuNode.waitForNewFinalization();
+
+    tekuNode.waitForNonDefaultExecutionPayload();
+    tekuNode.waitForNewFinalization();
+  }
+
+  private TekuNode.Config configureTekuNode(final TekuNode.Config node, final int genesisTime) {
+    return node.withNetwork(NETWORK_NAME)
+        .withBellatrixEpoch(UInt64.ONE)
+        .withTotalTerminalDifficulty(UInt64.valueOf(10001).toString())
+        .withGenesisTime(genesisTime);
+  }
+
+  private BesuNode.Config configureBesuNode(BesuNode.Config config) {
+    return config
+        .withRpcHttpApi(new String[] {"ETH,NET,WEB3,ENGINE"})
+        .withDefaultEngineRpcHttpPort()
+        .withEngineHostAllowList(new String[] {"*"})
+        .withMergeSupport(true)
```

Comment:
I wonder if we should just automatically set all of these as part of the `withMergeSupport(true)` call to keep the acceptance test classes abstracted away from the besu details a bit.

Reply:
Good idea. I have move these to `withMergeSupport(true)`.